"`#![feature]` may not be used on the stable release channel" when consuming crate

```
   Compiling swc_ecma_parser v0.66.5
error[E0554]: `#![feature]` may not be used on the stable release channel
   --> /Users/niklas/.cargo/registry/src/github.com-1ecc6299db9ec823/swc_ecma_parser-0.66.5/src/lib.rs:108:1
    |
108 | #![feature(bench_black_box)]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/ecmascript/parser/Cargo.toml
+++ b/ecmascript/parser/Cargo.toml
@@ -7,7 +7,7 @@ include = ["Cargo.toml", "src/**/*.rs", "examples/**/*.rs"]
 license = "Apache-2.0/MIT"
 name = "swc_ecma_parser"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.66.5"
+version = "0.66.6"
 
 [package.metadata.docs.rs]
 all-features = true

--- a/ecmascript/parser/src/lib.rs
+++ b/ecmascript/parser/src/lib.rs
@@ -103,9 +103,9 @@
 //!
 //! [tc39/test262]:https://github.com/tc39/test262
 
+#![cfg_attr(test, feature(bench_black_box))]
 #![cfg_attr(test, feature(test))]
 #![deny(unused)]
-#![feature(bench_black_box)]
 
 pub use self::{
     lexer::input::{Input, StringInput},

